package.json

```diff
@@ -23,21 +23,19 @@
   },
   "devDependencies": {
     "all-contributors-cli": "^3.0.0",
-    "ava": "^0.16.0",
     "babel-cli": "^6.7.7",
-    "babel-plugin-istanbul": "2.0.0",
+    "babel-jest": "15.0.0",
+    "babel-polyfill": "6.13.0",
```

Comment:
The fact that I just need these two babel deps installed and configure babel like normal to get ES6 working is pretty wonderful.